CONTRIBUTING.md

```diff
@@ -245,17 +245,10 @@ ICS adheres to the [trunk based development branching model](https://trunkbasedd
 
 ### Semantic Versioning
 
-ICS uses a variation of [semantic versioning](https://semver.org/).
+ICS follows [semantic versioning](https://semver.org), but with the following deviations (similar to [IBC-Go](https://github.com/cosmos/ibc-go/blob/main/RELEASES.md)):
 
-Note that state breaking changes are a subset of consensus breaking changes. Therefore we'll only refer to the latter when talking about versioning.
-
-ICS is a distributed, IBC based protocol in which multiple blockchains could be affected by a version bump. Therefore incrementing a MAJOR version number indicates that the PR updates, or is a breaking change to the way that the provider and consumer(s) communicate with one another over IBC. If a PR is consensus breaking to both the provider and consumer(s), then it requires a MAJOR version bump.
-
-Incrementing a MINOR version number indicates that a PR is only consensus breaking to the provider, or only to the consumers, where IBC communication remains unchanged.
-
-Incrementing a PATCH version number indicates that a PR is not consensus breaking to the provider or consumers. This could include node API changes, or other miscellaneous and often rare changes.
-
-Pure documentation, testing, and refactoring PRs do not require a version bump.
+- An API breaking change will result in an increase of the MAJOR version number (X.y.z | x > 0). 
```

Comment:
"API breaking change" is too vague 
```suggestion
- An API breaking change will result in an increase of the MAJOR version number (X.y.z | x > 0). This includes breaking changes to func signatures (conventional API breaking), adding/removing params, or anything that requires changes to an `app.go` external to the CCV module. 
```

Reply:
Also, one could interpret "API breaking" as node API. We should specify if the definition of API breaking should include changes to Txs or Queries

Reply:
Why would node API be different than API? The idea of semantic versioning is too inform the users of the changes in the code. It's clearly an overloaded concept, as you cannot use three numbers to identify changes in a blockchain protocol. So I don't think there is a perfect solution. Since ICS it's an IBC application, we should use the same semantic versioning as ibc-go. They manage to support 50+ chains with their semantic versioning, so we should be fine.